packages/cmrjs/tests/cmr-utils/test-reconcilecmrmetadata.js

```diff
@@ -359,7 +359,7 @@ test('publishUMMGJSON2CMR calls ingestUMMGranule with ummgMetadata via valid CMR
   await publishUMMGJSON2CMR(cmrPublishObject, new CmrFake());
 
   // Assert
-  t.true(ingestFake.calledOnceWithExactly(cmrPublishObject.metadataObject));
+  t.true(ingestFake.calledOnceWithExactly(cmrPublishObject.metadataObject, undefined));
```

Comment:
We have to assert `undefined` since no revisionId was passed to `publishUMMGJSON2CMR`?

Reply:
Yes since the test is checking for `calledOneWithExactly`